pkg/tbtc/dkg_submit_test.go

```diff
@@ -294,6 +294,10 @@ func TestSubmitResult_MemberSubmitsResult(t *testing.T) {
 	ctx, cancelCtx := context.WithCancel(context.Background())
 	defer cancelCtx()
 
+	if err = localChain.setDKGResultValidity(true); err != nil {
+		t.Fatal(err)
+	}
```

Comment:
Another - and maybe more intuitive? - option would be to treat the result as valid by default. We could set `dkgResultValid` in `Connect` / `ConnectWithKey`. Not a big deal though.